### ISSUE TITLE
Test snapshot packages for v0.11.0

### ISSUE THREAD

**Issue by amarthadan** [api3dao/airnode#1713]
Prior to the v0.11.0 release, the snapshot packages and corresponding Docker images need to be tested.

NPM packages:
```
@api3/airnode-abi@snapshot-v0.11.0
@api3/airnode-adapter@snapshot-v0.11.0
@api3/airnode-admin@snapshot-v0.11.0
@api3/airnode-deployer@snapshot-v0.11.0
@api3/airnode-examples@snapshot-v0.11.0
@api3/airnode-node@snapshot-v0.11.0
@api3/airnode-operation@snapshot-v0.11.0
@api3/airnode-protocol@snapshot-v0.11.0
@api3/airnode-utilities@snapshot-v0.11.0
@api3/airnode-validator@snapshot-v0.11.0
```

Docker images:
```
api3/airnode-deployer-dev:snapshot-v0.11.0
api3/airnode-client-dev:snapshot-v0.11.0
api3/airnode-admin-dev:snapshot-v0.11.0
```

**Comment by dcroote** [comment]
## v0.11 testing

tl;dr - everything worked 👍 

**A checked box indicates success.**

Test that npm packages `admin`, `deployer`, and `validator`, which have a `--help` CLI argument, are compiled correctly using `npx`

- [x] admin
  - [x] `generate-mnemonic`
- [x] deployer (tested more below)
- [x] validator
  - [x] Succeeds with `--config "packages/airnode-validator/test/fixtures/config.valid.json" --secrets "packages/airnode-validator/test/fixtures/secrets.valid.env"`

The following use the `coingecko-http-gateways` integration in `airnode-examples` on Ethereum sepolia (I initially began on goerli but the gas was a mess). I also modified the config to use two providers to see if #1718 happened to be reproduced, but I didn't encounter it.

Run Airnode client Docker image:

- [x] Testnet request
- [x] HTTP gateway request
- [x] Signed HTTP gateway request
- [x] Withdrawal request

Test AWS deployment with `deployer` Docker image:

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] `list`
  - [x] `info`
  - [x] `remove`

Test GCP deployment with `deployer` Docker image:

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] `list`
  - [x] `info`
  - [x] `remove-with-receipt` (differs from AWS to achieve additional command coverage)

## Failures

None.
